phac-nml/irida-next
# Disable automated pipelines

## What does this PR do and why?
Added a new status column to the automated pipelines table to show if the pipeline is enabled/disabled. 
Fixes #819.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/227c6f1a-27bf-4cf4-9de6-48fc6d49d5bb)

## How to set up and validate locally
1. Create an automated workflow execution within a project.
2. Remember the workflow name and version.
3. Modify the pipeline within the config/pipelines/pipelines.json file with `"automatable": false` or `"executable": false` or both or remove it entirely.
4. Restart irida-next.
5. Navigate to the project automation workflow executions page.
6. Verify the newly added automated workflow execution is disabled.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
